### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by Thi-Ponce**
### Capstone project:
- Create a simple webpage with a home and about page;
- Make sure it works for mobile (up to 768px);
- Make sure it works for desktop (base screen used 1262px);
- Meet the requirements expected;
- Loom link for the presentation: https://www.loom.com/share/37c8786a067349ee940345752908de6a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+
+- HTML, CSS, Js
+- VsCode
+## Authors
+
+👤 **Thiago Ponce**
+
+- GitHub: [@Thi-Ponce](https://github.com/Thi-Ponce)
+- Twitter: [@Thi_ponce](https://twitter.com/Thi_ponce)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/thiago-ponce)
```
Comment (rustam-tolipov) [inline README.md]: ## Good job so far 🙌

- [x] First of all, thank you for the great video presentation, it made it easy to review your code, don't forget to add Add descriptive README files to your project. Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+						<li class="nav-item">
+							<a href="#" class="nav-link ff-campaign">FanFest Campaign</a>
+						</li>
+					</ul>
+					<div class="hamburger">
+						<span class="line"></span>
+						<span class="line"></span>
+						<span class="line"></span>
+					</div>
+				</nav>
```
Comment (rustam-tolipov) [inline about.html]: ## Let's start improving your design 😉

- [x] There is one thing in your navigation that is missing. You forgot to add the black part at the top of the navigation which contains some links 🔗. Please kinds add that part and don't forget to hide it in the mobile version also
- [x] [OPTIONAL] Its optional but highly recommended. You are using tab size `8` but I suggest using `2`. Currently, it's making your code kinda bad looking and hard to read 👌

**Your design**
image

**Original design** 
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+			<a href="#" class="desktop-only">SEE THE WHOLE SCHEDULE!</a>
+		</div>
+	</section>
+	<section class="our-team">
+		<h2 class="team-title">Meet some of our team</h2>
+		<hr>
+		<div class="member-cards" id="team-members">
+			
+		</div>
+	</section>
```
Comment (rustam-tolipov) [inline index.html:132]: - [x] One thing is missing here is your footer. Let's copy the footer from about.html add here 📁

**Screenshot**
image

File: styles/about-style.css [hunk 4]
```diff
@@ -0,0 +1,504 @@
+* {
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  margin: 0;
+  padding: 0;
+}
```
Comment (rustam-tolipov) [inline styles/about-style.css:6]: ### I like your animation 😀. Keep on creating such great websites. Happy coding ☕

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @Thi-Ponce , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Dry code 💯
- Design is beautiful ✔


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (andersonlebon) [inline README.md]: - [x] As the previous reviewer said don't forget to add Add descriptive README files to your project. Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it with your project information. Kindly add some important sections in your Readme file like: ` Contributing`, `Acknowledgments`, ` License`, `Live Demo (if available)`, `Getting Started` referring yourself to [this template](https://github.com/microverseinc/readme-template).

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Thi-Ponce 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @Thi-Ponce 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes ♻️ 

- [x] The page styles are currently not loading in project, this issue may be because of the forward slash that is placed before the file name as seen here 👉🏻 `<link rel="stylesheet" href="/styles/about-style.css">`
- [ ] Another point is that the link to the live project is missing in the README. Please include it to allow users to have a feel of your project without looking through the code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**